### ISSUE TITLE
Empty observe array in mapping options creates observables from all properties

### ISSUE THREAD

**Issue by lwiseman** [SteveSanderson/knockout.mapping#204]
I expect the mapping plugin to create _no_ observables from the properties of the JS object, and instead, simply copy. See fiddle for a demonstration of the issue and a workaround:

http://jsfiddle.net/j6w18wba/

**Comment by crissdev** [comment]
@lwiseman I don't think this is likely to change. The library defaults other options like that as well - include, ignore etc. You might want to use `copy` option instead.
